subworkflows/nf-core/fasta_ltrharvest_ltrfinder_ltrretriever_lai/main.nf

```diff
@@ -0,0 +1,291 @@
+include { LTRHARVEST                } from '../../../modules/nf-core/ltrharvest/main'
+include { LTRFINDER                 } from '../../../modules/nf-core/ltrfinder/main'
+include { LTRRETRIEVER_LTRRETRIEVER } from '../../../modules/nf-core/ltrretriever/ltrretriever/main'
+include { CAT_CAT                   } from '../../../modules/nf-core/cat/cat/main'
+include { LTRRETRIEVER_LAI          } from '../../../modules/nf-core/ltrretriever/lai/main'
+
+workflow FASTA_LTRHARVEST_LTRFINDER_LTRRETRIEVER_LAI {
+
+    take:
+    ch_fasta                        // channel: [ val(meta), fasta ]
+    ch_monoploid_seqs               // channel: [ val(meta2), txt ]; Optional: Set to [] if not needed
```

Comment:
Rather than use `[]` for nothing, I think it would better to say use `Channel.empty()`. This might simplify some logic downstream